plugins/SitesManager/stylesheets/SitesManager.less

```
@@ -126,173 +126,263 @@ td.editable-site-field:hover {
 
 #site-without-data {
     background: @color-white;
+
     .top_controls {
         background: @color-white;
-    }
-}
 
-.site-without-data {
-    h3 {
-        font-weight: bold;
+        .quick-access {
+            display: none;
+        }
     }
 
-    p {
-        padding-bottom: 0.25rem;
+    .pageWrap {
+        max-width: 1168px; /* 1120px + padding */
+        margin: 0 auto;
+        padding: 0 24px;
     }
 
-    .no-data-screen-ul-tabs {
-        overflow: auto !important;height: auto !important;
-        li.col {
-            width: unset !important;
-            float: unset !important;
+    a#start-tracking-back {
+        color: @theme-color-text;
+        text-decoration: none;
+        margin-bottom: 16px;
+        line-height: 24px;
+        display: block;
+
+        [class^="icon-"], [class*=" icon-"] {
+            position: relative;
+            top: 1px;
         }
+    }
 
-        a:not(.active) {
-            text-decoration: none !important;
-            color: @color-black !important;
+    h1#start-tracking-data-header {
+        font-weight: bold;
+        font-size: 32px;
+        float: left;
+        margin-bottom: 32px;
+    }
 
-            &:hover {
-                background-color: @color-gray-bright !important;
-                opacity: unset;
+    #start-tracking-cta, #share-button {
+        float: right;
+
+        > a {
```

Could you please make this `> a, span > a {` ? Since the demo button is in a Vue template that uses a vue-entry, it's stuck in a containing element that throws off the styling. It would be very helpful to include anchor elements inside of spans as well.

---

Sure. Don't think that hurts.
```suggestion
        > a, > span > a {
```